eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -444,30 +454,31 @@ public void testCachedBadResponseRequiresFetch() {
     ConfigCacheFile cacheFile =
         new ConfigCacheFile(
             ApplicationProvider.getApplicationContext(), safeCacheKey(DUMMY_API_KEY));
-    cacheFile.setContents("{ invalid }");
+    cacheFile.setContents("ANYTHING IS INVALID HERE AS THE CONTENTS ARE SERIALIZED JAVA OBJECTS");
 
-    initClient(TEST_HOST, true, false, false, true, null, null, DUMMY_API_KEY);
+    initClient(TEST_HOST, true, false, false, true, null, null, DUMMY_API_KEY, false);
 
     double assignment = EppoClient.getInstance().getDoubleAssignment("numeric_flag", "alice", 0.0);
     assertEquals(3.1415926, assignment, 0.0000001);
   }
 
   @Test
-  public void testEmptyFlagsResponseRequiresFetch() {
+  public void testEmptyFlagsResponseRequiresFetch() throws IOException {
     // Populate the cache with a bad response
     ConfigCacheFile cacheFile =
         new ConfigCacheFile(
             ApplicationProvider.getApplicationContext(), safeCacheKey(DUMMY_API_KEY));
-    cacheFile.setContents("{\"flags\": {}}");
+    Configuration config = Configuration.emptyConfig();
+    cacheFile.getOutputStream().write(config.serializeFlagConfigToBytes());
```

Comment:
This fails with
```
cloud.eppo.android.EppoClientTest > testEmptyFlagsResponseRequiresFetch[emulator-5554 - 13] FAILED 
	java.lang.NullPointerException: Attempt to get length of null array
	at java.io.FileOutputStream.write(FileOutputStream.java:379)
```

because

`Configuration.emptyConfig()` sets `Configuration.flagConfigJson` to `null`, so `config.serializeFlagConfigToBytes()` returns `null`.